### ISSUE TITLE
Twitter::Error::Forbidden in twitter:update_lists

### ISSUE THREAD

**Issue by bitboxer** [hacken-in/website#503]
## Exception in Hacken.in

**Twitter::Error::Forbidden** in **twitter:update_lists**
You aren&#x27;t allowed to add members to this list

[View on Bugsnag](https://bugsnag.com/hacken-dot-in/hacken-dot-in/errors/54a228330d2531335063d468?event_id=54a2949c11a978ca23489726)

## Stacktrace

    lib/twitter_list_updater.rb:18 - block in add_missing_to_list
    lib/twitter_list_updater.rb:17 - each
    lib/twitter_list_updater.rb:17 - add_missing_to_list
    lib/twitter_list_updater.rb:11 - block in update
    lib/twitter_list_updater.rb:10 - update
    lib/tasks/twitter.rake:10 - block (2 levels) in &lt;top (required)&gt;

[View full stacktrace](https://bugsnag.com/hacken-dot-in/hacken-dot-in/errors/54a228330d2531335063d468?event_id=54a2949c11a978ca23489726)

**Comment by bascht** [comment]
Sorry – I mixed up #503 and #530 - although this one could be related.

**Comment by bascht** [comment]
@bitboxer when you added the credentials in #473 – did you check the twitter secrets?

They are empty for master:

```yaml:
  twitter_consumer_key: ''                                                              
  twitter_consumer_secret: ''                                                           
  twitter_access_token: ''                                                              
  twitter_access_token_secret: ''  
```

could this be the cause here?

**Comment by bitboxer** [comment]
Oh, shit. Right. I rechecked that for production. I will create a new twitter account for master and add the credentials.

**Comment by bitboxer** [comment]
This should be fixed now. There is another error on the staging that will be fixed when we pull in #573 and fix the unknown twitter handles on staging :)
